### ISSUE TITLE
Make facet panels non-collidable when background is off

### ISSUE THREAD

**Issue by shadowpanther** [Yellow-Dog-Man/Resonite-Issues#2242]
### Is your feature request related to a problem? Please describe.

I'm always frustrated when I'm only using one small facet, but the whole facet panel (and the other panels with no facets at all) takes my laser when it is on.

### Describe the solution you'd like

Only add collision on the hand panel when the background is on or when UI edit mode is on.
Similar to how view panels are only collidable on facets themselves and while editing.

### Describe alternatives you've considered

Not using facet panels or override-disabling all facet anchors except the ones with actual facets.

**Comment by Frooxius** [comment]
Why exactly is the collision bothering you on these?

The rationale for the view one is that it covers your view and is always visible.

But the hand ones are toggled and are generally on your hand where there shouldn't be other UI to interact with - or there might be and it might interfere.

I feel like I don't understand the rationale of making them passthrough. We can make the changes, but I want to understand the problem better.

**Comment by shadowpanther** [comment]
@Frooxius there are other interactive elements on the hand, like the tool shelf and some custom avatar systems. While panels are on (and invisible because background is off), it's quite hard to get to these.

**Comment by shiftyscales** [comment]
Yeah- I've actually ran into this issue too, @Frooxius. Historically for as long as I've used the platform, I've had my toolshelves on the back of my avatar's hands- in the same position the facet anchors now rest. For avatars I've not yet manually corrected, my toolshelves are not usable at the same time as the facet anchors.

I've had to compromise so far by repositioning the hand anchor off of the back of my hand off to the side of it. (To this end- I'd still like if #2123 could possibly be considered so I could have a single universal placement for these anchors across all of my avatars, past, present, and future- and I would consider as another suitable solution to my personal needs with this issue.)

Because the anchors take laser priority over everything else in the world- they can sometimes just get in the way of other things too I've found.

**Comment by shiftyscales** [comment]
Updated the title to be a bit more broad, rather than focusing on a specific request to the underlying issue.
